olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -500,17 +500,17 @@ spec:
                       - name: "IMAGE_KN_CLI_ARTIFACTS"
                         value: "registry.ci.openshift.org/openshift/knative-v0.20.0:kn-cli-artifacts"
                       - name: "KAFKA_IMAGE_kafka-controller-manager__manager"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-source-controller"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.20.0:knative-eventing-kafka-source-controller"
                       - name: "KAFKA_IMAGE_KAFKA_RA_IMAGE"
-                        value: "registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-receive-adapter"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.20.0:knative-eventing-kafka-receive-adapter"
```

Comment:
@lberk this adds back the "old" `...svc...` registry

Reply:
aha! thank you!